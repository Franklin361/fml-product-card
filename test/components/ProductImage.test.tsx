import React from 'react';
import render from 'react-test-renderer';
import { ProductImage, ProductCard } from '../../src/components';

import { product2 } from '../data/products';

describe('Product Image', () => {
    test('deberia mostrar el componente', () => {
        const wrapper = render.create(
            <ProductImage image={product2.image}/>
        )

        expect(wrapper.toJSON()).toMatchSnapshot();
    })
    
    test('Debe de mostrar el componente con la url de la imagen', () => {
        
        const wrapper = render.create(
            <ProductCard product={product2}>
                {
                    () => (
                        <ProductImage/>
                    )
                }
            </ProductCard>
        )

        expect(wrapper.toJSON()).toMatchSnapshot();

    })
})
