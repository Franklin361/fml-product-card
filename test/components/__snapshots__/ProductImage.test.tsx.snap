// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Product Image Debe de mostrar el componente con la url de la imagen 1`] = `
<div
  className="productCard undefined"
>
  <img
    alt="Coffe Mug"
    className="productImg undefined"
    src="../image.jpg"
  />
</div>
`;

exports[`Product Image deberia mostrar el componente 1`] = `
<img
  alt="Coffe Mug"
  className="productImg undefined"
  src="../image.jpg"
/>
`;
